### ISSUE TITLE
Unable to move items from crafter into player inventory

### ISSUE THREAD

**Issue by Meryl** [McJty/RFTools#2153]
Issue description: I'm playing FTB OceanBlock 1.11.0, and am unable to move items from the crafter into my inventory, but I _can_ move them into my toolbar.

Steps to reproduce:

1. Open a crafter.
2. Insert an item.
3. Try taking it out and putting it in your inventory.
4. Try putting it in your toolbar instead.

Versions:
* Minecraft: 1.16.5
* Forge: 36.2.20
* CompatLayer (only if on Minecraft 1.10 or 1.11): n/a
* McJtyLib: 5.1.2
* RFTools: Base 2.1.1 / Builder 3.1.4 / Power 3.0.11 / Storage 2.0.15 / Utility 3.1.6

Possibly incompatible mods and versions: Not using optifine, using default mods and settings for FTB OceanBlock 1.11.0

Relevant logs, if any: n/a

**Comment by McJty** [comment]
I cannot reproduce this. What exactly are you doing? Precise steps. Preferably with screenshots

**Comment by Meryl** [comment]
[This](https://user-images.githubusercontent.com/1791364/147245751-04bb85ab-495d-4125-8ce0-88c6d7ccb7ab.png) is what the crafter looks like. The selected recipe is configured as shown, the other recipes all use 'Int' to keep the essences inside.

[Here's](https://user-images.githubusercontent.com/1791364/147246187-4adaba12-0049-4a58-95ce-375e47c13ffc.mp4) a video of trying to put the insanium essence into a random slot in the inventory. As you can see (the cursor flashes yellow when I click), it won't go in. It does go in the toolbar.

**Comment by McJty** [comment]
Weird. I think that must be some other mod doing this. I cannot reproduce it

**Comment by Meryl** [comment]
I did some more testing, it has to do with the 'remembered' items and the problem only actually occurs in 26 of the player's inventory slots (which just so happens to be the number of input slots on the crafter). Items also _do_ work in slots which the corresponding input slot has that item as a remembered item.

[Here's](https://user-images.githubusercontent.com/1791364/147248638-06142e97-d8c7-4b50-b83f-133cf150b2fb.mp4) a video demonstration, as you can see the stone does go in the first slot (no corresponding slot in the input inventory), it doesn't go into the next five slots (item doesn't match the remembered items in the first 5 slots of the input inventory) but it does go into the rest of the slots (item _does_ match the remembered items).

**Comment by McJty** [comment]
No, still can't reproduce it. Doing exactly the same as what you're doing. I do think some other mod is messing things up there

**Comment by McJty** [comment]
No, still can't reproduce it. Doing exactly the same as what you're doing. I do think some other mod is messing things up there

**Comment by Meryl** [comment]
[This](https://user-images.githubusercontent.com/1791364/147250206-0804c4af-1480-43ad-a2d3-e8d8ca7ef2d3.mp4) is with just the RFTools mods installed.

Log: [debug.log](https://github.com/McJtyMods/RFTools/files/7769647/debug.log) / [latest.log](https://github.com/McJtyMods/RFTools/files/7769654/latest.log)
Installed mods: [image](https://user-images.githubusercontent.com/1791364/147250412-96bcd420-c892-4d39-9014-b752f3119199.png)
World save: [New World.zip](https://github.com/McJtyMods/RFTools/files/7769657/New.World.zip)

**Comment by McJty** [comment]
Well weird. No clue what's going on then
